docs/howto_guides/replacing_asdfs.rst

```diff
@@ -0,0 +1,72 @@
+.. _dkist:howto-guide:replacing-asdfs:
+
+Replacing Previously Downloaded ASDF Files
+==========================================
+
+The DKIST Data Center will occasionally update the ASDF files for all the datasets it stores, due to changes or corrections to the metadata.
+This can lead to two possible problems.
+First, when you download ASDF metadata files from the Data Center those files might be expecting a newer version of the Python tools.
+Second, ASDF files you have downloaded previously might become outdated and need to be re-downloaded.
+
+
+Re-downloading ASDF files
+-------------------------
+
+You should periodically re-download your local ASDF files to keep up to date with changes to the dataset metadata.
+To do so, you can use the `overwrite` keyword argument when downloading an ASDF file.
+For example, to force a refresh of your local copy of the sample data:
+
+.. code-block:: python
+
+    from dkist.data.sample import download_all_sample_data
+
+    download_all_sample_data(overwrite=True) # doctest: +REMOTE_DATA
+
+
+`Fido.fetch` also takes the `overwrite` argument.
+
+Occasionally, the naming convention for the ASDF files might also change, meaning that the usual checks to stop you downloading a dataset you already have locally will fail and you will end up with two (likely identical) metadata files.
```

Comment:
I would not expect this to change again in the future, you should rewrite this in the past tense.